test/specs/Loader.spec.js

```diff
@@ -109,6 +109,11 @@ beforeEach(function() {
     gaExportKmlProvider.downloadKmlUrl = gaGlobalOptions.apiUrl + '/downloadkml';
   });
 
+  module(function(gaExportGlStyleProvider, gaGlobalOptions) {
+    gaExportGlStyleProvider.downloadUrl = gaGlobalOptions.apiUrl +
+        '/downloadkml';
```

Comment:
can't you download directly from public URLs? IE doesn't support 
what if we change the Content-Disposition in S3, it could be downloaded directly without going through the API no?
we can leave it for now but we need to address this in the future.

Reply:
I guess download service was there for a reason. I don't remember why. Whatever it's not part of this project.